include/interface/blas1_interface.h

```diff
@@ -136,9 +136,9 @@ typename sb_handle_t::event_t _asum(
  * \brief Prototype for the internal implementation of the ASUM operation. See
  * documentation in the blas1_interface.hpp file for details.
  */
-template <int localSize, int localMemSize, typename sb_handle_t,
-          typename container_0_t, typename container_1_t, typename index_t,
-          typename increment_t>
+template <int localSize, int localMemSize, bool usmManagedMem = false,
```

Comment:
Can you add a short doxygen documentation for `usmManagedMem` and why it's needed everywhere it's used?

Reply:
I added a small description for it in `src/interface/blas1_interface.hpp` along with the rest of the documentation.
Changes in f89fd4b55da6f4f92bb72c5db8f3b097cca7b7bd